cluster/vagrant/provision-master.sh

```diff
@@ -93,20 +93,19 @@ grains:
   node_ip: '$(echo "$MASTER_IP" | sed -e "s/'/''/g")'
   publicAddressOverride: '$(echo "$MASTER_IP" | sed -e "s/'/''/g")'
   network_mode: openvswitch
-  networkInterfaceName: eth1
+  networkInterfaceName: '$(echo "$NETWORK_IF_NAME" | sed -e "s/'/''/g")'
   api_servers: '$(echo "$MASTER_IP" | sed -e "s/'/''/g")'
   cloud: vagrant
   roles:
     - kubernetes-master
   runtime_config: '$(echo "$RUNTIME_CONFIG" | sed -e "s/'/''/g")'
   docker_opts: '$(echo "$DOCKER_OPTS" | sed -e "s/'/''/g")'
-  master_extra_sans: '$(echo "$MASTER_EXTRA_SANS" | sed -e "s/'/''/g")'  
+  master_extra_sans: '$(echo "$MASTER_EXTRA_SANS" | sed -e "s/'/''/g")'
 EOF
 
 mkdir -p /srv/salt-overlay/pillar
 cat <<EOF >/srv/salt-overlay/pillar/cluster-params.sls
   service_cluster_ip_range: '$(echo "$SERVICE_CLUSTER_IP_RANGE" | sed -e "s/'/''/g")'
-  cert_ip: '$(echo "$MASTER_IP" | sed -e "s/'/''/g")'
```

Comment:
This should not be removed, otherwise we will not generate certificates properly.

See how this is needed:
https://github.com/fredjean/kubernetes/blob/fix-vagrant-vmware/cluster/saltbase/salt/generate-cert/init.sls#L18

To do this:
https://github.com/fredjean/kubernetes/blob/fix-vagrant-vmware/cluster/saltbase/salt/generate-cert/init.sls#L24